diagnostic.md

```diff
@@ -73,14 +96,26 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request "POST" "http://www.example.com/donuts/" \
+  --header "Content-Type: application/json" \
+  --data $'{
+    "donut": {
+      "name": $NAME,
+      "price": $PRICE
+    }
+  }'
 ```
 
 Write an AJAX request to create a single donut on the server using JSON. Please
 do not use `data = getFormFields(form)` instead write out the data object.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = function (event) {
+  event.preventDefault();
+  libraryApi.create(event.target)
+    .done(ui.logResponseBody)
+    .fail(ui.logRequestError);
+};
```

Comment:
1/2. I see what you wanted to do here but this is not an ajax request, the libraryApu.create() function that this calls would be the ajax request